packages/shared/providers.yaml

```diff
@@ -1545,6 +1545,15 @@ luma:
             method: GET
             endpoint: /public/v1/user/get-self
     docs: https://docs.nango.dev/integrations/all/luma
+listmonk:
+    categories:
+        - marketing
+    auth_mode: BASIC
+    proxy:
+        base_url: https://${connectionConfig.domain}/api
+        headers:
+            'Content-Type': 'application/json'
```

Comment:
```suggestion
            content-type: application/json
```

Reply:
Should i omit the header as hassan suggested or use samuels suggestion?

Reply:
@samuelandert, please use Samuel's suggestion.